common/OpTestOpenBMC.py

```diff
@@ -447,22 +448,23 @@ def clear_sel(self):
     '''
     set boot device to setup
     curl -c cjar -b cjar -k -H "Content-Type: application/json" -d "{\"data\": \"Setup\"}" -X PUT
-    https://bmc/org/openbmc/settings/host0/attr/boot_flags
+    https://bmc/xyz/openbmc_project/control/host0/boot/attr/bootsource
     '''
     def set_bootdev_to_setup(self):
         data = '\'{"data": \"Setup\"}\''
-        obj = "/org/openbmc/settings/host0/attr/boot_flags"
+        obj = "xyz/openbmc_project/control/host0/boot/attr/bootsource"
         self.curl.feed_data(dbus_object=obj, operation='rw', command="PUT", data=data)
         self.curl.run()
 
     '''
     set boot device to default
-    curl -c cjar -b cjar -k -H "Content-Type: application/json" -d "{\"data\": \"Default\"}" -X PUT
-    https://bmc/org/openbmc/settings/host0/attr/boot_flags
+    curl -c cjar -b cjar -k -H "Content-Type: application/json" -d
+    "{\"data\": \"xyz.openbmc_project.Control.Boot.Source.Sources.Default\"}" -X PUT
+    https://bmc/xyz/openbmc_project/control/host0/boot/attr/bootsource
     '''
     def set_bootdev_to_none(self):
-        data = '\'{\"data\": \"Default\"}\''
-        obj = "/org/openbmc/settings/host0/attr/boot_flags"
+        data = '\'{\"data\": \"xyz.openbmc_project.Control.Boot.Source.Sources.Default\"}\''
+        obj = "xyz/openbmc_project/control/host0/boot/attr/bootsource"
```

Comment:
Add these both `set_bootdev_to_setup` and `set_bootdev_to_none` functions in  RestAPI testcase testcases/testRestAPI.py . This is the standalone testcase we want actually to verify quickly REST API things on OpenBMC in-addition to using in regular IPL path.